.github/workflows/scripts.yml

```diff
@@ -0,0 +1,23 @@
+# workflow for communication with eps1lon/mui-scripts-incubator
+# These scripts are experimental. We have a very "dumb" workflow here that is only responsible for triggering
+# actions in eps1lon/mui-scripts-incubator. Since those actions are experimental they'll likely break
+# and we don't want to disrupt the normal repo workflow.
+name: experimental
+on: status
+jobs:
+  build:
+    if: github.event.state == 'success' && github.event.sender.login == 'netlify[bot]' && github.event.description == 'Deploy preview ready!'
```

Comment:
Tested in https://github.com/eps1lon/material-ui-playroom/runs/492647605?check_suite_focus=true